src/sass/static/pages.scss

```diff
@@ -1,5 +1,60 @@
 // styling for static pages that are too short or simple to need their own styling page
 
+.affiliates_faq {
+    h3 {
+        color: $COLOR_ORANGE;
+
+        &:not(:first-of-type) {
+            margin-top: 3rem;
+        }
+    }
+}
+
+.affiliates {
+    h2 {
+        margin-top: 30px;
+    }
+    #commission_tabs {
+        margin-bottom: 0 !important;
+    }
+    .margin-bottom-50 {
+        margin-bottom: 50px;
+    }
+    .fixed-height-img {
+        height: 50px;
+    }
+    .faq-box {
+        padding: 1rem;
+    }
+    .box-row {
+        border-bottom: 1px solid $COLOR_GRAY;
+
+        .box {
+            padding: 2rem;
+        }
+        &-top {
+            .box {
+                padding-top: 1rem;
+            }
+        }
+        &-bottom {
+            border: none;
+
+            .box {
+                padding-bottom: 0;
+            }
+        }
+        .box:not(:last-child) {
+            border-right: 1px solid $COLOR_GRAY;
+        }
+        @media screen and (max-width: 480px) {
+            &, .box {
+                border: none;
+            }
+        }
+    }
+}
```

Comment:
on the other hand, this seems long enough to warrant being in its own affiliate.scss file, but it's up to you, since https://github.com/binary-com/binary-static/blob/master/src/sass/static/pages.scss#L343 seems to be way too long and it's somehow in this file...